pkg/apis/kops/networking.go

```diff
@@ -183,6 +183,11 @@ type CalicoNetworkingSpec struct {
 	TyphaPrometheusMetricsPort int32 `json:"typhaPrometheusMetricsPort,omitempty"`
 	// TyphaReplicas is the number of replicas of Typha to deploy
 	TyphaReplicas int32 `json:"typhaReplicas,omitempty"`
+	// VXLANMode is the encapsulation mode to use for the default Calico IPv4 pool created at start
+	// up, determining when to use VXLAN encapsulation, conveyed to the "calico-node" daemon
+	// container via the CALICO_IPV4POOL_VXLAN environment variable.
```

Comment:
This is a run-on sentence. It should be broken up into multiple sentences and/or simplified.

Document the possible values?